jbpm-workitems/jbpm-workitems-archetype/src/main/resources/archetype-resources/jbpm-workitems-__rootArtifactId__/src/main/resources/kie-ddtemplate.st

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<deployment-descriptor xsi:schemaLocation="http://www.jboss.org/jbpm deployment-descriptor.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <persistence-unit>org.jbpm.domain</persistence-unit>
+    <audit-persistence-unit>org.jbpm.domain</audit-persistence-unit>
+    <audit-mode>JPA</audit-mode>
+    <persistence-mode>JPA</persistence-mode>
+    <runtime-strategy>SINGLETON</runtime-strategy>
```

Comment:
Does this mean runtime strategy is not configurable? Or it does not matter?

Reply:
That's just the default that the workbench uses as well. Users are free to change before copy/paste into their project if they wish, or they can just take the handlers part out of it too.